BraintreeCore/src/main/java/com/braintreepayments/api/core/PostalAddress.kt

```diff
@@ -9,18 +9,16 @@ import android.os.Parcelable
  */
 @Parcelize
 data class PostalAddress(
-    var recipientName: String?,
-    var phoneNumber: String?,
-    var streetAddress: String?,
-    var extendedAddress: String?,
-    var locality: String?,
-    var region: String?,
-    var postalCode: String?,
-    var sortingCode: String?,
-    var countryCodeAlpha2: String?) : Parcelable {
-
-    constructor() : this(null, null, null, null, null, null, null, null, null)
```

Comment:
It'll be great if we don't need this constructor any more. Usually we include this one when we want Java interop without the `@JvmOverloads` annotation.